README.md

```diff
@@ -201,3 +200,21 @@ request.AddHeader("Content-Type", "application/json");
 request.AddParameter("undefined", "{\n\t\"to\" : \"+250781234567\", \n\t\"text\" : \"Hello from Pindo\",\n\t\"sender\" : \"Pindo\"\n}", ParameterType.RequestBody);
 IRestResponse response = client.Execute(request);
 ```
+
+```ruby
+# ruby
+
+require 'net/http'
+require 'json'
+
+uri = URI('http://api.pindo.io/v1/sms/')
```

Comment:
I think `require "uri"` is needed.

Reply:
yeah if used in rails, you don't need to require it but it is needed in plain ruby. will add that too